tensorflow/core/kernels/resize_nearest_neighbor_op_gpu.cu.cc

```diff
@@ -172,22 +172,24 @@ struct ResizeNearestNeighbor<GPUDevice, T, half_pixel_centers, align_corners> {
     const int output_size = batch_size * out_height * out_width * channels;
     if (output_size == 0) return true;
 
-    GpuLaunchConfig config = GetCudaLaunchConfig(output_size, d);
+    GpuLaunchConfig config = GetGpuLaunchConfig(output_size, d);
     if (half_pixel_centers) {
-      TF_CHECK_OK(CudaLaunchKernel(
+      TF_CHECK_OK(GpuLaunchKernel(
           ResizeNearestNeighborNHWC<T>, config.block_count,
           config.thread_per_block, 0, d.stream(), output_size, input.data(),
-          in_height, in_width, channels, out_height, out_width, height_scale,
-          width_scale, output.data()));
+          static_cast<int>(in_height), static_cast<int>(in_width), channels,
```

Comment:
Maybe change the type of these and cast them during value initialization?

Reply:
done